src/javascript/base.js

```diff
@@ -0,0 +1,14 @@
+function client_secret_store () {
```

Comment:
make 2 functions, one which stores stuff in LocalStorage, one for SessionStorage, for that function have params as key, value so that those storage functions can be re-used.

@safurazaheer and you can think on it to make a common way to store/retrieve stuff from Local/Session Storage